test/functional/http/get/transactions.js

```diff
@@ -383,6 +398,52 @@ describe('GET /api/transactions', () => {
 			});
 		});
 
+		describe('senderIdOrRecipientId', () => {
+			it('using invalid senderIdOrRecipientId should fail', () => {
+				return transactionsEndpoint
+					.makeRequest({ senderIdOrRecipientId: '' }, 400)
+					.then(res => {
+						expectSwaggerParamError(res, 'senderIdOrRecipientId');
+					});
+			});
+
+			it('using one senderIdOrRecipientId should return incoming and outgoing transaction of an account', () => {
+				var sender = [];
+				var recipient = [];
+				var accountId = account.address;
+				return transactionsEndpoint
+					.makeRequest({ senderIdOrRecipientId: accountId }, 200)
+					.then(res => {
+						expect(res.body.data).to.not.empty;
+						res.body.data.map(transaction => {
+							if (accountId === transaction.recipientId) {
+								recipient.push(transaction);
+							} else if (accountId === transaction.senderId) {
+								sender.push(transaction);
+							}
+						});
+						expect(sender).to.have.length(1);
+						expect(recipient).to.have.length(1);
```

Comment:
The transactions you posted are deterministic and you know there are only two such transactions. So having more clear expectations will be nice. 

```
expect(res.body.data).to.not.empty;
expect(res.body.data[0].recipientId).to.be.eql(accountId);
expect(res.body.data[1].senderId).to.be.eql(accountId);
```

Notice how clean three line expectations look like and easy to understand.

Reply:
It is true there are only these 2 transactions, but is are they returned always the same chronological order inside of `res.body.data`?
If not this expectations could fail randomly.

Reply:
Logically there should be one specific order whatever it would be. There can't be random order.

Reply:
ok, will change it then!